app/src/main/java/org/p2p/wallet/root/RootActivity.kt

```diff
@@ -104,10 +103,6 @@ class RootActivity :
         toast(message)
     }
 
-    override fun onBackPressed() {
-        popBackStack()
```

Comment:
sure that we need to remove this ?

Reply:
again, try to debug all onBackDispatchers outside this branch and check that onBackDispatchers are not working

Reply:
This will break the navigation. 
This is the behavior for the fragments that don't have dispatcher callbacks.